registrations/tasks.py

```diff
@@ -273,8 +274,10 @@ def create_subscriptionrequests(self, registration):
         weeks = 1  # default week number
 
         # . calculate weeks along
-        if (registration.reg_type == "momconnect_prebirth" and
-           registration.source.authority not in ["hw_partial", "patient"]):
+        if (registration.reg_type in [
+                "momconnect_prebirth", "whatsapp_prebirth"] and
+                registration.source.authority not in [
+                    "hw_partial", "patient"]):
```

Comment:
@rudigiesler so the tests pass but if I leave it out entirely, the weeks always defaults to 1 -- which doen't seem right. Unless that's the case for all public registrations as momconnect_prebirth too?